crates/bevy_log/src/lib.rs

```diff
@@ -111,6 +103,7 @@ impl Default for LogSettings {
 }
 
 impl Plugin for LogPlugin {
+    #[allow(unused_variables)]
```

Comment:
Should we just be using `_app` here?

Reply:
no, `app` is used if feature `tracing-chrome` is enabled...